docs/quick-start.md

```diff
@@ -17,14 +17,14 @@ First, install and setup cri-resource-manager.
 #### CentOS, Fedora and SUSE
 
 ```
-CRIRM_VERSION=0.4.0
+CRIRM_VERSION=0.4.1
```

Comment:
A similar comment applies here than for the [related PR to the main branch](https://github.com/intel/cri-resource-manager/pull/583#discussion_r547535884)...